### PR TITLE
Feature/parse level select

### PR DESCRIPTION
**Pull request by michaelDemertzi**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def build_game_controller(width, height, level_path='levels/example.txt') -> Bel
     view = ConsoleView(model, model_vars, writer)
     return BellhopController(model, view)
 
-def build_visuals_controller(width, height, level_path='levels/example.txt') -> BellhopController:
+def build_visuals_controller(width, height, level_path) -> BellhopController:
```
Comment (thbenschop) [inline main.py]: whoops these args in the functions of main should have types

File: view/console_view.py [hunk 2]
```diff
@@ -79,6 +81,9 @@ def get_elevator_with_people(self) -> str:
         elevator_with_people = self._put_people_in_elevator(merged_people)
         return elevator_with_people
 
+    def get_stats(self) -> str:
+        return "<< Move: {}>>".format(self._bellhop_model.get_move_number())
```
Comment (thbenschop) [inline view/console_view.py:85]: no tips??

Comment (screwed99) [reply view/console_view.py:85]: tips should be moved to and read from model

Comment (michaelDemertzi) [reply view/console_view.py:85]: yeah but do we actually want a secondary score apart from moves? the tips was mostly a joke. If we want to keep it I'll move to model

Comment (thbenschop) [reply view/console_view.py:85]: I thought that efficiency etc were going to be in it too, only saying it score and putting it in not view. it's a good placeholder

File: main.py (hunk 1 above)
Comment (michaelDemertzi) [reply main.py]: done

File: view/console_view.py (hunk 2 above)
Comment (michaelDemertzi) [reply view/console_view.py:85]: oh @thbenschop yeah no tips for same reason I said before. It's generated at completion by the view only. If we actually want it (?) throughout I will move the code and show there. 

I'm in favor of the only obj being to try to minimize moves for now but lmk

**Review by thbenschop — APPROVED** [review]

**Pull request merged**